_conferences/pucelatechday-2020.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Pucela Tech Day 2020"
```

Comment:
Hi Diana!
We have just updated the README indicating to avoid the year in the name, it's not necessary due the date it's already there. Do you mind if I ask you to remove it before to merge this PR?

Thanks in advance Diana!

Reply:
OMG! I read it and I forgot, sorry. 😐 
I fix right now.
Thanks!

Reply:
OK, Sorry! :-) I didn’t understand this, but I don’t know how to modify this file now. :/ 
I see you've already changed it, haven't you? I'm sorry, I barely use Github and I'm not sure how PR works here.

Reply:
Don't you worry @dianaaceves!
Yep, I've already update the file because it's an event we wanted to have listed!
Anyway, if you need any help with PR just let me know and I can help you for whatever you need!
Thanks for adding this amazing conference to the website!